src/lib.rs

```diff
@@ -1,23 +1,27 @@
-use tracing::{
-    subscriber::{set_global_default, SetGlobalDefaultError}, Level,
+use tracing::subscriber::{set_global_default, SetGlobalDefaultError};
+
+pub use tracing::{
+    debug, debug_span, error, error_span, info, info_span, span, trace, trace_span, warn,
+    warn_span, Level,
 };
 
 /// Initialize `dioxus-logger` with a specified max filter.
 /// Generally it is best to initialize the logger before launching your Dioxus app.
 /// Works on Web, Desktop, Fullstack, and Liveview.
-/// 
+///
 /// # Example
-/// 
-/// ```rust,ignore
+///
+/// ```rust
 /// use dioxus::prelude::*;
-/// use tracing::{Level, info};
-/// 
+/// use dioxus_logger::{Level, info};
+///
 /// fn main() {
 ///     dioxus_logger::init(Level::INFO).expect("logger failed to init");
-///     launch(App);
+///     // launch app:
+///     // e.g. launch(App);
 /// }
-/// 
-/// #[component]
+///
+///
```

Comment:
Shouldn't you at least add `#[allow(non_snake_case)]` ?

Reply:
I was going to re-add the component macro. Removed it because doc test was acting weird but it wasn't the problem.